libs/ledger-live-common/src/apps/polyfill.ts

```diff
@@ -67,28 +67,39 @@ export function declareDep(name: string, dep: string): void {
   ["Volta", "Ethereum"],
   ["Yearn", "Ethereum"],
 ].forEach(([name, dep]) => declareDep(name, dep));
-export const getDependencies = (appName: string): string[] =>
-  directDep[appName] || [];
+
+const versionBasedWhitelistDependencies = {
+  Bitcoin: "2.1.0",
+};
+
+export const getDependencies = (
+  appName: string,
+  appVersion?: string
+): string[] => {
+  const maybeDirectDep = directDep[appName];
+  if (!maybeDirectDep) return [];
+
+  // Nb Starting on version 2.1.0 Bitcoin family apps no longer depend on
```

Comment:
this comment belong to line 72 I think? before the `Bicoin: "2.1.0"`

Reply:
Bitcoin is the only currency currently doing this standalone approach, but I guess maybe it will extend to other currencies 🤔. I'm ok with having it there for now and removing it once it's more than just Bitcoin. If it annoys you a lot I can change it though 🙏🏼